### PR TITLE
added section on not looking at solutions until you are done

### PR DESCRIPTION
**Pull request by CatQueenCodes**
Added a section about not looking at student solutions. I added it just before the first project and am going to add a one liner before each of the foundations projects along the lines of "Remember don't look at the project solutions until you are finished, ask for help in the Discord server if you get stuck!". I will add this after this PR is merged (if you chose to merge it). that way I can link back to this section since it should have a link to this specific addition that I can use.

Please Ping me here or discord if this is merged so I can add a line to the other foundations projects 😸
@marvingay redone

### CODE REVIEW

File: foundations/the_front_end/project_html_css.md [hunk 1]
```diff
@@ -46,6 +46,25 @@ If you have entered `git push origin main` and typed out your username and passw
 
 Okay, that's enough Git for the moment -- time to actually build stuff!
 
+### Warning About Looking at Other People's Finished Project Code Before You have Completed Your Own
+
+All of our projects have a "Student Solutions" section at the bottom for people to post their completed code and live websites for the projects. These are here for you to compare your **completed** code to and see what other routes people have chosen to take with their project code and designs.
```
Comment (marvingay) [inline foundations/the_front_end/project_html_css.md:51]: This is bad.

**Review by marvingay — CHANGES_REQUESTED** [review]
This is a test.

File: foundations/the_front_end/project_html_css.md (hunk 1 above)
Comment (CatQueenCodes) [reply foundations/the_front_end/project_html_css.md:51]: Is that a test too, or is the wording actually bad and I need to change it

File: foundations/the_front_end/project_html_css.md [hunk 2]
```diff
@@ -46,6 +46,25 @@ If you have entered `git push origin main` and typed out your username and passw
 
 Okay, that's enough Git for the moment -- time to actually build stuff!
 
+### Warning About Looking at Other People's Finished Project Code Before You have Completed Your Own
+
+All of our projects have a "Student Solutions" section at the bottom for people to post their completed code and live websites for the projects. These are here for you to compare your **completed** code to and see what other routes people have chosen to take with their project code and designs. 
+
+We know it may be tempting to look at the solutions when you get stuck or don't know where to start, but **don't** do it! Here are some reasons why:
+
+- All of the submitted projects are other *student* solutions. This means they are written by other people who are also still learning and may not necessarily be good examples to go off of.
```
Comment (marvingay) [inline foundations/the_front_end/project_html_css.md:55]: and may not necessarily be representative of "best practice" or quality code.

File: foundations/the_front_end/project_html_css.md [hunk 3]
```diff
@@ -46,6 +46,25 @@ If you have entered `git push origin main` and typed out your username and passw
...
+- Looking at solutions robs you of the chance to develop your problem solving and researching skills.
+- There are a ton of different ways to get the same end result, it is not "one size fits all".
```
Comment (marvingay) [inline foundations/the_front_end/project_html_css.md:57]: "There are many different ways to achieve the same end result,"

File: foundations/the_front_end/project_html_css.md [hunk 4]
```diff
@@ -46,6 +46,25 @@ If you have entered `git push origin main` and typed out your username and passw
...
+- In the real world, there will not always "solutions" for you to look at and you will have to come up with your own.
```
Comment (marvingay) [inline foundations/the_front_end/project_html_css.md:58]: ...there will not always **be** 

Grammar error. 

This bullet point could also afford to be under the one for "Looking at solutions"

**Review by marvingay — CHANGES_REQUESTED** [review]
Left some comments just to formalize some of the verbiage and address a minor grammar issue.

**Comment by CatQueenCodes** [comment]
Updated it a few days ago but forgot to comment!

**Review by marvingay — APPROVED** [review]
Thanks for the changes! Looks good to me.

**Pull request merged**